setup/config/setupJest.js

```diff
@@ -11,7 +11,7 @@ import ReactPiwik from 'react-piwik';
 import crypto from 'crypto';
 import ReactRouterDom from 'react-router-dom';
 import * as ReactRedux from 'react-redux';
-import { deepMergeObj } from 'src/utils/helpers';
+import lodashMerge from 'lodash.merge';
```

Comment:
maybe we could use our sdk utils to remove as much deps as possible? https://github.com/LiskHQ/lisk-sdk/blob/daf69a202de7fc95a9bffac142ec16e455c03bb9/elements/lisk-utils/src/objects/merge_deep.ts#L23

example: https://github.com/LiskHQ/lisk-sdk/blob/5f88e62e59e2a982723a371c757e818c4401837b/commander/src/bootstrapping/commands/start.ts#L170

Reply:
Loadash merge is already a dependency. It didn't introduce it. Changing it causes some tests to fail because of the differences in their functionality.